drafts/draft-ioamteam-ippm-ioam-direct-export.xml

```diff
@@ -225,10 +225,12 @@
 	  Pre-allocated Trace Option, Incremental Trace 
 	  Option, Proof of Transit (POT) Option, and Edge-to-Edge Option.</t>
 	  
-	  <t>This document defines a new IOAM option type (also known as an IOAM type) 
-	  called the Direct Export (DEX) option. This option
-	  is used as a trigger for IOAM nodes to export IOAM
-	  data to a collector.</t>
+	  <t>This document defines a new IOAM option type (also known as an IOAM 
+	  type) called the Direct Export (DEX) option. This option is used as a 
+	  trigger for IOAM nodes to export IOAM data to a receiving entity. The
+	  "receiving entity" in this context can be, for example, an external 
+	  collector, analyzer, controller, encapsulating node, decapsulating node, 
```

Comment:
Loopback does not currently work with Direct Export since there is no information added directly to the data packet. At least as currently defined, loopback uses the IOAM data format, not the export data format. Encapsulating node should be dropped from this list. It might come back in future proposals, but since this list is just examples and is not meant to be comprehensive, it should be fine without encapsulating node.